interface.go

```diff
@@ -63,6 +63,8 @@ type Config struct {
 	// If this field is not set, the Dial functions will return only when the connection is forward secure.
 	// Callbacks have to be thread-safe, since they might be called in separate goroutines.
 	ConnState ConnStateCallback
+	// KeepAlive defines whether keep-alive is enabled from the client to server.
```

Comment:
I was taking about the default value in that comment. There are applications where the distinction between client and server is negligible (think peer to peer), and where it's not clear that the "server" should not ping. I'd like to keep this option for the future.